evaluation/poets/run.sh

```diff
@@ -0,0 +1,5 @@
+for d in scripts/* ; do
+    echo "$d"
+    /pash/pa.sh $d > /dev/null
```

Comment:
I would replace `/pash` with `$PASH_TOP` so that this works for machines where pash is not under `/pash`